src/utils/network/session-core.js

```diff
@@ -764,3 +764,22 @@ export const sendMessageTTL = async (session, messageObject) => {
     throw new Error('Message failed to send. Reason: Inbox does not exist for sender or recipient');
   }
 };
+
+/**
+ * Function that creates an outbox container in the user's Pod when logging in for
+ * the first time
+ *
+ * @memberof utils
+ * @function createOutbox
+ * @param {Session} session - Solid's Session Object {@link Session}
+ * @returns {Promise} Promise - Generates an outbox for Pod upon log in if
+ * user's Pod does not have the an outbox to begin with
+ */
+
+export const createOutbox = async (session) => {
+  const outboxContainerUrl = getContainerUrl(session, 'Outbox', 'self-fetch');
+  await createContainerAt(outboxContainerUrl, { fetch: session.fetch });
+
+  // Generate ACL file for container
+  await createDocAclForUser(session, outboxContainerUrl);
```

Comment:
Will be replaced with the new `setDocAclForUser` function in PR #183. Will merge this as is if merged before #183 and after review.